src/components/LogInModal/LogInModal.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import styles from './LogInModal.module.css';
```

Comment:
do you use ```React```?

Reply:
yes. in:
 const handleClick = (event: React.MouseEvent<HTMLDivElement>) => {
    if (event.target === event.currentTarget) {
       onCloseModal();
    }
 };